static-instrumenter/agent-instrumenter/build.gradle.kts

```diff
@@ -1,18 +1,134 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
 plugins {
   id("otel.java-conventions")
+  id("com.github.johnrengelman.shadow") version "7.1.2"
+  id("org.unbroken-dome.test-sets") version "4.0.0"
 }
 
 description = "OpenTelemetry Java Static Instrumentation Agent"
 
+val javaagentLibs: Configuration by configurations.creating {
+  isCanBeResolved = true
+  isCanBeConsumed = false
+}
+
+val bootstrapLibs: Configuration by configurations.creating
+configurations.getByName("implementation").extendsFrom(bootstrapLibs)
+
+val javaagent: Configuration by configurations.creating
+configurations.getByName("implementation").extendsFrom(javaagent)
+
 dependencies {
+  annotationProcessor("com.google.auto.service:auto-service")
+  compileOnly("com.google.auto.service:auto-service")
+
   implementation("org.slf4j:slf4j-api")
   runtimeOnly("org.slf4j:slf4j-simple")
+
+  // TODO: remove snapshot once new agent is released
+  javaagent("io.opentelemetry.javaagent:opentelemetry-javaagent:1.14.0-SNAPSHOT")
```

Comment:
The versions should already be set by the `dependencyManagement` -- can you try removing it and checking if it works?

Reply:
adding to `DEPENDENCIES` doesn't work so I left it as it is - I'll try placing it there again after the new release